dev/CommonStyles/FlyoutPresenter_themeresources.xaml

```diff
@@ -54,13 +49,12 @@
             <Setter.Value>
                 <ControlTemplate TargetType="FlyoutPresenter">
                     <Border
-                        Background="{TemplateBinding Background}"
-                        contract7Present:BackgroundSizing="OuterBorderEdge"
+                        Background="{TemplateBinding Background}"                        
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
-                        Padding="{ThemeResource FlyoutBorderThemePadding}">
+                        contract7Present:BackgroundSizing="OuterBorderEdge"
+                        Padding="{TemplateBinding Padding}">
```

Comment:
`Padding="{TemplateBinding Padding}"` will push the scroll viewer in away from the edge of the flyout which I don't think we want.

Reply:
The new design for the scroll viewer has rounded corners, so it may be intended

Reply:
I've updated the way Mark suggested. We want scrollviewer to be flush, but then the content has padding (which a dev can customize via template).